cmd/minikube/cmd/start.go

```diff
@@ -75,11 +75,20 @@ func runStart(cmd *cobra.Command, args []string) {
 	api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 	defer api.Close()
 
+	diskSize := viper.GetString(humanReadableDiskSize)
+	diskSizeMB := calculateDiskSizeInMB(diskSize)
+	minimumDiskSizeMB := calculateDiskSizeInMB(constants.MinimumDiskSize)
```

Comment:
Could you make this the constant (probably has to be a var, but can still live in constants.go?